src/vs/workbench/electron-sandbox/desktop.contribution.ts

```diff
@@ -360,6 +360,10 @@ import { MAX_ZOOM_LEVEL, MIN_ZOOM_LEVEL } from 'vs/platform/window/electron-sand
 				type: 'boolean',
 				description: localize('argv.disableLcdText', 'Disables LCD font antialiasing.')
 			},
+			'proxy-bypass-list': {
+				type: 'string',
+				description: localize('argv.proxyBypassList', 'Bypass any specified proxy for the given semi-colon-separated list of hosts.')
```

Comment:
Maybe add example of host format accepted in this arg, can use the example from https://github.com/electron/electron/blob/main/docs/api/command-line-switches.md#--proxy-bypass-listhosts